AutomatedTesting/Gem/PythonTests/Physics/tests/tick/Tick_InterpolatedRigidBodyMotionIsSmooth.py

```diff
@@ -0,0 +1,96 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Test Case Title : Verify that a rigid body with "Interpolate motion" option selected moves smoothly.
+"""
+
+
+# fmt: off
+class Tests():
+    create_entity          = ("Created test entity",                           "Failed to create test entity")
+    rigid_body_added       = ("Added PhysX Rigid Body component",              "Failed to add PhysX Rigid Body component")
+    rigid_body_smooth      = ("Rigid body motion passed smoothness threshold", "Failed to meet smoothness threshold for rigid body motion")
+# fmt: on
+
+
+def Tick_InterpolatedRigidBodyMotionIsSmooth():
+    """
+    Summary:
+     Create entity with PhysX Rigid Body component and turn on the Interpolate motion setting.
+     Verify that the position of the rigid body varies smoothly with time.
+
+    Expected Behavior:
+     1) The motion of the rigid body under gravity is a smooth curve, rather than an erratic/jittery movement.
+
+    Test Steps:
+     1) Load the empty level
+     2) Create an entity
+     3) Add rigid body component
+     4) Enter game mode and collect data for the rigid body's z co-ordinate and the time values for a series of frames
+     5) Check if the motion of the rigid body was sufficiently smooth
+
+    :return: None
+    """
+    # imports
+    import os
+    import azlmbr.legacy.general as general
+    import azlmbr.math as math
+    from editor_python_test_tools.editor_entity_utils import EditorEntity as Entity
+    from editor_python_test_tools.utils import Report
+    from editor_python_test_tools.utils import TestHelper as helper
+    from editor_python_test_tools.asset_utils import Asset
+    import numpy as np
+
+    # constants
+    COEFFICIENT_OF_DETERMINATION_THRESHOLD = 1 - 1e-4 # curves with values below this are not considered sufficiently smooth
+
+    helper.init_idle()
+    # 1) Load the empty level
+    helper.open_level("Physics", "Base")
+
+    # 2) Create an entity
+    test_entity = Entity.create_editor_entity("test_entity")
+    Report.result(Tests.create_entity, test_entity.id.IsValid())
+
+    azlmbr.components.TransformBus(
+        azlmbr.bus.Event, "SetWorldTranslation", test_entity.id, math.Vector3(0.0, 0.0, 0.0))
+
+    # 3) Add rigid body component
+    rigid_body_component = test_entity.add_component("PhysX Rigid Body")
+    rigid_body_component.set_component_property_value("Configuration|Interpolate motion", True)
+    azlmbr.physics.RigidBodyRequestBus(azlmbr.bus.Event, "SetLinearDamping", test_entity.id, 0.0)
+    Report.result(Tests.rigid_body_added, test_entity.has_component("PhysX Rigid Body"))
+
+    # 4) Enter game mode and collect data for the rigid body's z co-ordinate and the time values for a series of frames
+    t = []
+    z = []
+    general.enter_game_mode()
```

Comment:
assert that we are in game mode just in case

Reply:
Added result reporting for entering and exiting.